sections/placeholders.md

```diff
@@ -0,0 +1,117 @@
+# Placeholder Resources
+
+##### Endpoint: `/api/v1/placeholder_resources`
+
+Placeholder resources can be used for temporary resourcing on projects where the final resources are not yet known. Placeholders behave quite similarly to users, with a few differences discussed below.
+
+## List Placeholder Resources (index)
+
+##### Optional Parameters:
+
+| **Parameter** | **Description** |
+| ------------- | --------------- |
+| fields | A comma separated list of additional fields to include in the response [ "assignments", "custom_field_values"] |
```

Comment:
Yes, it is true. It would be all assignments and custom field values for that placeholder. This is the same pattern the users API uses. Take a look at the RABL template for /api/v1/placeholder_resources/show.rabl - for assignments and custom field values, it redirects to those RABL templates.